roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+openshift_logging_eventrouter_image_prefix: gcr.io/heptio-images/
+openshift_logging_eventrouter_image_version: v0.1
+openshift_logging_eventrouter_replicas: 1
+openshift_logging_eventrouter_sink: stdout
+openshift_logging_eventrouter_nodeselector: ""
+openshift_logging_eventrouter_cpu_limit: 100m
+openshift_logging_eventrouter_memory_limit: 128Mi
+openshift_logging_eventrouter_namespace: default
```

Comment:
Do we have any understanding if this should deploy here or the logging namespace?

Reply:
That is what was accepted in [the proposal doc](https://github.com/openshift/origin-aggregated-logging/blob/master/docs/proposals/kube_events_design_doc.md)

Reply:
right.  This was so the events ended up in the operations index.

Reply:
Please do not deploy anything to default.  @sdodson nothing new should be going into default at this point, no matter what kind of project.

Everything deployed by openshift-ansible that is infrastructure component must go into an `openshift-*` namespace, no exceptions.  This allows us to control evolution of these.

Nothing should default to "default".

Reply:
@smarterclayton This is the only way, at least initially, these events end up in an 'operations' index in Elastic.  Also, are you suggesting we modify the default for logging to be something like 'openshift-infra-logging' or 'openshift-logging'?  We never agreed on what it should be way back when  logging was first released.